service/flags.go

```diff
@@ -22,13 +22,15 @@ import (
 )
 
 const (
-	configFlag = "config"
-	setFlag    = "set"
+	configFlag    = "config"
+	setFlag       = "set"
+	hotReloadFlag = "enable-hot-reload"
```

Comment:
"config-autoreload"?

I don't like necessarily the "hot" part of the name. "auto" sounds better to me.

Also important to mention what is reloaded :)

Reply:
Hm, how about just `--watch-config`? That's consistent with our internal terminology, and reasonably descriptive. I'm fine with `--enable-auto-reload` as well though.

Reply:
`--watch-config` is fine